en:
   external_accounts:
     connected_as: "Connected as: %{name}"
     disconnect: Disconnect
     connect: Connect
     connected_to:
       success: Account from %{site} connected successfully.
     connection_error: |
       There was an error during the authentication. Please try
       again later.
   dac:
     code: RA
     submit: Submit
     cancel: Cancel
     invalid_code: Invalid code
